notes.md

```
@@ -22,3 +22,14 @@ TODO: orders with invoice and details
     Add license code colum to order items table. 
 
 TODO: add review button to the review tab on the product. so first or create to update existing reviews
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)